pkg/scorecard_test.go

```diff
@@ -341,3 +346,65 @@ func TestExperimentalRunProbes(t *testing.T) {
 		})
 	}
 }
+
+func Test_findConfigFile(t *testing.T) {
+	t.Parallel()
+
+	//nolint:govet
+	tests := []struct {
+		locs      []string
+		desc      string
+		wantFound bool
+	}{
+		{
+			desc:      "scorecard.yml exists",
+			locs:      []string{"scorecard.yml"},
+			wantFound: true,
+		},
+		{
+			desc:      ".scorecard.yml exists",
+			locs:      []string{".scorecard.yml"},
+			wantFound: true,
+		},
+		{
+			desc:      ".github/scorecard.yml exists",
+			locs:      []string{".github/scorecard.yml"},
+			wantFound: true,
+		},
+		{
+			desc:      "multiple configs exist",
+			locs:      []string{"scorecard.yml", ".github/scorecard.yml"},
+			wantFound: true,
+		},
```

Comment:
do we want to test precedence at all? or nah?

Reply:
Added a test for precedence